### DISCUSSION TITLE
app died 2 y ago

### DISCUSSION THREAD

**Discussion by ribosee** [FSystem88/spymer#60] [General]
put this work on archivied and tell people  in Readme file how dead ur spymer, maybe they didn't lose 3 hours like me. Have a nice day bye

**Comment by FSystem88** [comment]
> put this work on archivied and tell people in Readme file how dead ur spymer, maybe they didn't lose 3 hours like me. Have a nice day bye

the description says where to follow the news on this project, where it says in the post that the project is dead! 

to download, run and test the program, 5 minutes is enough, go to the TG channel - a minute. If you have already spent several hours - your problem!
